File: sigma_marketing_app/sigma_marketing/lib/data/models/request/registration/new_profile.dart
class NewProfile {
  String firstName;
  String lastName;
  String email;
  String password;
  String gender;
  DateTime birthDate;
  String phoneNumber;

  NewProfile({
    required this.firstName,
    required this.lastName,
    required this.email,
    required this.password,
    required this.gender,
    required this.birthDate,
    required this.phoneNumber,
  });

  Map<String, dynamic> toJson() {
    return {
      'firstName': firstName,
      'lastName': lastName,
      'email': email,
      'password': password,
      'gender': gender,
      'birthDate': birthDate.toIso8601String(),
      'phoneNumber': phoneNumber,
    };
  }
}

File: sigma_marketing_api/SigmaMarketing/SigmaMarketing.Model/Models/Request/NewProfile.cs
namespace SigmaMarketing.Model.Models.Request
{
    public class NewProfile
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public string Gender { get; set; }

        public DateTime BirthDate { get; set; }
        public string PhoneNumber { get; set; }
    }
}
